cmd/tracee-ebpf/internal/config/config.go

```diff
@@ -0,0 +1,80 @@
+package config
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"os"
+)
+
+type TraceeEbpfCliConfig struct {
```

Comment:
did we test it? I am not sure the unmarshal at line 39 will work if the struct is not provided with json labels.

Reply:
I tested it, it works, but it needs to correspond one to one (not case sensitive) with the json keys, so for the long term it would be a bit better with tags if we want to change the struct names (structure itself would still need to be mapped 1 to 1).

Reply:
Added json tags, done